docs/api/plot.rst

```diff
@@ -152,3 +152,11 @@ validation_curve
 cooks_distance
 --------------
 .. autofunction:: sklearn_evaluation.plot.cooks_distance
+
+report_evaluation
+-----------------
+.. autofunction:: sklearn_evaluation.models.evaluate_model
```

Comment:
maybe I'm not recalling correctly, but I think we decided to move this to the report module?

so `from sklearn_evaluation.report import evaluate_model`?

Reply:
yes. changed it.